src/R/send-charts.R

```diff
@@ -0,0 +1,203 @@
+library(dplyr)
+library(ggplot2)
+library(reshape2)
+library(stringr)
+
+### Variables for all charts ###
+
+df_historical <- read.csv("../../target/historic-data.csv")
```

Comment:
I suspect that this "probably" would work from someone else's machine, but I would prefer if we could add absolute paths. Perhaps an RProject is the solution?

Reply:
I'm fairly confident this is the cleanest way to work because this Rscript is explicitly called from the location /src/R/ by Clojure which sets the workspace for the script there. I considered a few ways of doing this and this seemed like the safest and simplest option, while keeping this production code in a src directory.